go/logic/applier.go

```diff
@@ -49,6 +49,15 @@ func newDmlBuildResultError(err error) *dmlBuildResult {
 	}
 }
 
+func (this *Applier) setOptimizer(tx *gosql.Tx) error {
+	if this.migrationContext.OptimizerSwitch == "" {
+		return nil
+	}
+	optimizerString := fmt.Sprintf("set session optimizer_switch=%q", this.migrationContext.OptimizerSwitch)
```

Comment:
```suggestion
	optimizerString := fmt.Sprintf("SET SESSION optimizer_switch=%q", this.migrationContext.OptimizerSwitch)
```
I suggest we use capital `SET SESSION` to match the rest of `applier.go`